third_party/OWNERS

```diff
@@ -2,6 +2,7 @@
 
 reviewers:
   - dims
+  - justaugustus
```

Comment:
@thockin @smarterclayton

Reply:
https://github.com/kubernetes/kubernetes/pull/93888 adds the dep-approvers alias as an approver for this package... I actually wonder if that should be used for the reviewer list for this one as well

Reply:
Removed that bit from the PR.